discovery-infra/download_logs.py

```diff
@@ -102,6 +103,9 @@ def download_logs(client: InventoryClient, cluster: dict, dest: str, must_gather
     try:
         write_metadata_file(client, cluster, os.path.join(output_folder, 'metdata.json'))
 
+        with suppress(AssertionError, ConnectionError):
```

Comment:
Why do we suppress those?

Reply:
Keep running and download everything possible, even if something had failed.